RefApp/AssetsViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  AssetsViewController.swift
+//  RefApp
+//
+//  Created by Karl Holmlöv on 10/31/18.
+//  Copyright © 2018 emp. All rights reserved.
+//
+
+import UIKit
+import ExposurePlayback
+import Player
+import Exposure
+
+class AssetsViewController: UITableViewController {
+
+    var myIndex = 0
+    var assets = [Asset]()
+    var environment: Environment!
+    var sessionToken: SessionToken?
+    var query = "assetType=MOVIE"
+    var endpoint = "/content/asset"
+    var method: HTTPMethod = .get
+
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        guard let baseUrl = UserDefaults.standard.string(forKey: "expUrl"), let customer = UserDefaults.standard.string(forKey: "customer"), let buisnessUnit = UserDefaults.standard.string(forKey: "buisnessUnit")
+        else {
+            return
+        }
+        
+        environment = Environment(baseUrl: baseUrl, customer: customer, businessUnit: buisnessUnit)
+        print(environment)
+        
+        loadAssets(query: self.query, environment: self.environment, endpoint: self.endpoint, method: self.method)
+        assets.forEach({ Asset in
+            print(Asset)
+        })
+        print(assets.count)
+        tableView.reloadData()
+        
+    }
+    func loadAssets(query: String, environment: Environment, endpoint: String, method: HTTPMethod) {
+        print("Loading assets")
+        self.assets.removeAll()
+        ExposureApi<AssetList>(environment: environment,
+                               endpoint: endpoint,
+                               query: query,
+                               method: method)
+            .request()
+            .validate()
+            .response { [weak self] in
+                guard let `self` = self else {
+                    return
+                }
+                if let value = $0.value?.items {
+                    value.forEach{
+                        self.assets.append($0)
```

Comment:
since we clear out all assets before each load, this could be replaced by a simple assignment

self.assets = $0.value?.items ?? []